atomic_reactor/schemas/container.json

```diff
@@ -165,6 +165,10 @@
         "demo"
       ]
     },
+    "set_release_env": {
+      "type": "string",
+      "description": "Name of an environmental variable to set in the Dockerfile file with the release number"
```

Comment:
environmental variable → environment variable

Reply:
Fixed and examples added.